src/Compilers/VisualBasic/Portable/Syntax/SyntaxFactory.vb

```diff
@@ -1111,7 +1111,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             For Each err In lastToken.GetDiagnostics()
                 Select Case DirectCast(err.Code, ERRID)
                     Case ERRID.ERR_UnterminatedStringLiteral
-                        If Parser.CheckFeatureAvailability(languageVersion, Feature.MultilineStringLiterals) Then
+                        If InternalSyntax.IsAvailable(Feature.MultilineStringLiterals, options) Then
```

Comment:
Can this be invoked as an extension too? `Feature.MultilineStringLiterals.IsAvailable(options)`

Reply:
Yep it can missed it when moved the extension methods.